frontend/test/__support__/ui.tsx

```diff
@@ -386,7 +393,7 @@ export const mockGetBoundingClientRect = (options: Partial<DOMRect> = {}) => {
         right: 0,
         x: 0,
         y: 0,
-        toJSON: () => {},
+        toJSON: () => { },
```

Comment:
This is a function that returns undefined, but could be misconstrued as a function that returns an empty object `{}`.

Prefer the following
```suggestion
        toJSON: () => undefined,
```

Or if you did want the object:
```suggestion
        toJSON: () => ({}),
```

Reply:
it's been a result of merge conflict, but I fixed it